[kf5-kauth] build failure

```
Installing kde-mac/kde/kolourpaint dependency: KDE-mac/kde/kf5-kauth
==> Downloading https://download.kde.org/stable/frameworks/5.42/kauth-5.42.0.tar.xz
Already downloaded: /Users/diver/Library/Caches/Homebrew/kf5-kauth-5.42.0.tar.xz
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/kf5-kauth/5
==> make install
Last 15 lines from /Users/diver/Library/Logs/Homebrew/kf5-kauth/02.make:
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKF5Auth_EXPORTS -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -DQT_WIDGETS_LIB -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/KF5Auth_autogen/include -iframework /usr/local/opt/qt/lib -isystem /usr/local/opt/qt/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt/./mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5 -isystem /usr/local/opt/qt/lib/QtWidgets.framework/Headers -isystem /usr/local/opt/qt/lib/QtGui.framework/Headers -isystem /System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -isystem /usr/local/opt/qt/lib/QtDBus.framework/Headers  -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Wvla -Wdate-time  -DNDEBUG -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -fPIC -std=gnu++11 -o CMakeFiles/KF5Auth.dir/kauthdebug.cpp.o -c /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/kauthdebug.cpp
[ 86%] Building CXX object src/CMakeFiles/KF5Auth.dir/backends/fake/FakeBackend.cpp.o
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKF5Auth_EXPORTS -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -DQT_WIDGETS_LIB -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/KF5Auth_autogen/include -iframework /usr/local/opt/qt/lib -isystem /usr/local/opt/qt/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt/./mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5 -isystem /usr/local/opt/qt/lib/QtWidgets.framework/Headers -isystem /usr/local/opt/qt/lib/QtGui.framework/Headers -isystem /System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -isystem /usr/local/opt/qt/lib/QtDBus.framework/Headers  -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Wvla -Wdate-time  -DNDEBUG -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -fPIC -std=gnu++11 -o CMakeFiles/KF5Auth.dir/backends/fake/FakeBackend.cpp.o -c /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src/backends/fake/FakeBackend.cpp
[ 86%] Building CXX object src/CMakeFiles/KF5Auth.dir/backends/fakehelper/FakeHelperProxy.cpp.o
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKF5Auth_EXPORTS -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -DQT_WIDGETS_LIB -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/KF5Auth_autogen/include -iframework /usr/local/opt/qt/lib -isystem /usr/local/opt/qt/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt/./mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5 -isystem /usr/local/opt/qt/lib/QtWidgets.framework/Headers -isystem /usr/local/opt/qt/lib/QtGui.framework/Headers -isystem /System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -isystem /usr/local/opt/qt/lib/QtDBus.framework/Headers  -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Wvla -Wdate-time  -DNDEBUG -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -fPIC -std=gnu++11 -o CMakeFiles/KF5Auth.dir/backends/fakehelper/FakeHelperProxy.cpp.o -c /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src/backends/fakehelper/FakeHelperProxy.cpp
[ 87%] Building CXX object src/CMakeFiles/KF5Auth.dir/ECMQmLoader-kauth5_qt.cpp.o
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKF5Auth_EXPORTS -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -DQT_WIDGETS_LIB -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/KF5Auth_autogen/include -iframework /usr/local/opt/qt/lib -isystem /usr/local/opt/qt/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt/./mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5 -isystem /usr/local/opt/qt/lib/QtWidgets.framework/Headers -isystem /usr/local/opt/qt/lib/QtGui.framework/Headers -isystem /System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -isystem /usr/local/opt/qt/lib/QtDBus.framework/Headers  -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Wvla -Wdate-time  -DNDEBUG -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -fPIC -std=gnu++11 -o CMakeFiles/KF5Auth.dir/ECMQmLoader-kauth5_qt.cpp.o -c /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/ECMQmLoader-kauth5_qt.cpp
[ 88%] Building CXX object src/CMakeFiles/KF5Auth.dir/KF5Auth_autogen/mocs_compilation.cpp.o
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -DKCOREADDONS_LIB -DKF5Auth_EXPORTS -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_MAC_USE_COCOA -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_SIGNALS_SLOTS_KEYWORDS -DQT_NO_URL_CAST_FROM_STRING -DQT_USE_FAST_OPERATOR_PLUS -DQT_USE_QSTRINGBUILDER -DQT_WIDGETS_LIB -D_DARWIN_C_SOURCE -D_LARGEFILE64_SOURCE -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/src -I/tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/KF5Auth_autogen/include -iframework /usr/local/opt/qt/lib -isystem /usr/local/opt/qt/lib/QtCore.framework/Headers -isystem /usr/local/opt/qt/./mkspecs/macx-clang -isystem /usr/local/include/KF5/KCoreAddons -isystem /usr/local/include/KF5 -isystem /usr/local/opt/qt/lib/QtWidgets.framework/Headers -isystem /usr/local/opt/qt/lib/QtGui.framework/Headers -isystem /System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -isystem /usr/local/opt/qt/lib/QtDBus.framework/Headers  -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Wvla -Wdate-time  -DNDEBUG -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -fPIC -std=gnu++11 -o CMakeFiles/KF5Auth.dir/KF5Auth_autogen/mocs_compilation.cpp.o -c /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src/KF5Auth_autogen/mocs_compilation.cpp
[ 88%] Linking CXX shared library ../bin/libKF5Auth.dylib
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Cellar/cmake/3.10.1/bin/cmake -E cmake_link_script CMakeFiles/KF5Auth.dir/link.txt --verbose=1
/usr/local/Homebrew/Library/Homebrew/shims/super/clang++  -std=c++0x -fno-operator-names -fno-exceptions -Wno-gnu-zero-variadic-macro-arguments -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Wvla -Wdate-time  -DNDEBUG -dynamiclib -Wl,-headerpad_max_install_names -single_module -multiply_defined suppress  -compatibility_version 5.0.0 -current_version 5.42.0 -o ../bin/libKF5Auth.5.42.0.dylib -install_name @rpath/libKF5Auth.5.dylib CMakeFiles/KF5Auth.dir/kauthaction.cpp.o CMakeFiles/KF5Auth.dir/kauthactionreply.cpp.o CMakeFiles/KF5Auth.dir/kauthexecutejob.cpp.o CMakeFiles/KF5Auth.dir/kauthobjectdecorator.cpp.o CMakeFiles/KF5Auth.dir/AuthBackend.cpp.o CMakeFiles/KF5Auth.dir/BackendsManager.cpp.o CMakeFiles/KF5Auth.dir/HelperProxy.cpp.o CMakeFiles/KF5Auth.dir/kauthhelpersupport.cpp.o CMakeFiles/KF5Auth.dir/kauthdebug.cpp.o CMakeFiles/KF5Auth.dir/backends/fake/FakeBackend.cpp.o CMakeFiles/KF5Auth.dir/backends/fakehelper/FakeHelperProxy.cpp.o CMakeFiles/KF5Auth.dir/ECMQmLoader-kauth5_qt.cpp.o CMakeFiles/KF5Auth.dir/KF5Auth_autogen/mocs_compilation.cpp.o /usr/local/lib/libKF5CoreAddons.5.42.0.dylib /usr/local/opt/qt/lib/QtWidgets.framework/QtWidgets /usr/local/opt/qt/lib/QtDBus.framework/QtDBus /usr/local/opt/qt/lib/QtGui.framework/QtGui /usr/local/opt/qt/lib/QtCore.framework/QtCore
cd /tmp/kf5-kauth-20180115-69651-w2imzx/kauth-5.42.0/build/src && /usr/local/Cellar/cmake/3.10.1/bin/cmake -E cmake_symlink_library ../bin/libKF5Auth.5.42.0.dylib ../bin/libKF5Auth.5.dylib ../bin/libKF5Auth.dylib
[ 88%] Built target KF5Auth
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/KDE-mac/homebrew-kde/issues
```

Building manually produces this error:

```
[ 79%] Linking CXX executable ../bin/kauth-policy-gen
Undefined symbols for architecture x86_64:
  "output(QList<Action>, QMap<QString, QString> const&)", referenced from:
      _main in policy-gen.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [bin/kauth-policy-gen] Error 1
make[1]: *** [src/CMakeFiles/kauth-policy-gen.dir/all] Error 2
make: *** [all] Error 2
```

--- a/kf5-kauth.rb
+++ b/kf5-kauth.rb
@@ -15,6 +15,8 @@ class Kf5Kauth < Formula
   depends_on "gpgme"
   depends_on "KDE-mac/kde/kf5-kcoreaddons"
 
+  patch :DATA
+
   def install
     args = std_cmake_args
     args << "-DBUILD_TESTING=OFF"
@@ -35,3 +37,47 @@ class Kf5Kauth < Formula
      EOS
   end
 end
+
+__END__
+
+diff --git a/src/backends/fake/kauth-policy-gen-polkit.cpp b/src/backends/fake/kauth-policy-gen-polkit.cpp
+index d103414..5575620 100644
+--- a/src/backends/fake/kauth-policy-gen-polkit.cpp
++++ b/src/backends/fake/kauth-policy-gen-polkit.cpp
+@@ -38,7 +38,7 @@ const char policy_tag[] = ""
+ 
+ const char dent[] = "   ";
+ 
+-void output(QList<Action> actions, QMap<QString, QString> domain)
++void output(QList<Action> actions, const QMap<QString, QString> &domain)
+ {
+     Q_UNUSED(domain)
+ 
+diff --git a/src/backends/mac/kauth-policy-gen-mac.cpp b/src/backends/mac/kauth-policy-gen-mac.cpp
+index e146c46..ccfc0e7 100644
+--- a/src/backends/mac/kauth-policy-gen-mac.cpp
++++ b/src/backends/mac/kauth-policy-gen-mac.cpp
+@@ -26,7 +26,7 @@
+ 
+ using namespace std;
+ 
+-void output(QList<Action> actions, QMap<QString, QString> domain)
++void output(QList<Action> actions, const QMap<QString, QString> &domain)
+ {
+     AuthorizationRef auth;
+     AuthorizationCreate(NULL, kAuthorizationEmptyEnvironment, kAuthorizationFlagDefaults, &auth);
+diff --git a/src/backends/policykit/kauth-policy-gen-polkit.cpp b/src/backends/policykit/kauth-policy-gen-polkit.cpp
+index 4f29cb0..c215275 100644
+--- a/src/backends/policykit/kauth-policy-gen-polkit.cpp
++++ b/src/backends/policykit/kauth-policy-gen-polkit.cpp
+@@ -39,7 +39,7 @@ const char policy_tag[] = ""
+ 
+ const char dent[] = "   ";
+ 
+-void output(QList<Action> actions, QHash<QString, QString> domain)
++void output(QList<Action> actions, const QMap<QString, QString> &domain)
+ {
+     QTextStream out(stdout);
+     out.setCodec("UTF-8");
+
+

--- a/kf5-kauth.rb
+++ b/kf5-kauth.rb
@@ -3,7 +3,7 @@ class Kf5Kauth < Formula
   homepage "https://www.kde.org"
   url "https://download.kde.org/stable/frameworks/5.42/kauth-5.42.0.tar.xz"
   sha256 "91ebf3554551c3815e89e53e577e42d7cc3ac1f4966215fbbc93a60ca1587812"
-  revision 1
+  revision 2
 
   head "git://anongit.kde.org/kauth.git"
 

